client/my-sites/post-type-list/post-actions-ellipsis-menu/qrcode.jsx

```diff
@@ -0,0 +1,65 @@
+					fill-color="#000000"
```

Comment:
Nit: is there a chance this might inadvertently take precedence over the color scheme of the app? Should we find a different place to style it, one that's more compatible with the color scheming capabilities of Calypso?

Reply:
Just removed the fill-color prop. This icon is using the same properties as `gridicon` so it is using the same color scheme.

We are expecting to introduce a new qr-code icon on gridicon soon, but we think this is okay as a workaround for now.